app/security/Roles.scala

```diff
@@ -121,59 +121,60 @@ object Roles {
-      activeUserWithActiveApp(user) && statusIn(user)(WITHDRAWN)
+      statusIn(user)(WITHDRAWN)
   }
 
   object OnlineTestInvitedRole extends CsrAuthorization {
     override def isAuthorized(user: CachedData)(implicit request: RequestHeader, lang: Lang) =
-      activeUserWithActiveApp(user) && statusIn(user)(PHASE1_TESTS)
+      activeUserWithActiveApp(user) && (statusIn(user)(PHASE1_TESTS) && isPhase1TestsInvited(user))
   }
 
   object OnlineTestExpiredRole extends CsrAuthorization {
     override def isAuthorized(user: CachedData)(implicit request: RequestHeader, lang: Lang) =
-      activeUserWithActiveApp(user) && statusIn(user)(PHASE1_TESTS) && isTestExpired(user)
+      activeUserWithActiveApp(user) && isPhase1TestsExpired(user)
   }
 
   object Phase1TestFailedRole extends CsrAuthorization {
     override def isAuthorized(user: CachedData)(implicit request: RequestHeader, lang: Lang) =
-      activeUserWithActiveApp(user) && statusIn(user)(PHASE1_TESTS_FAILED)
+      activeUserWithActiveApp(user) && isPhase1TestsFailed(user)
   }
 
   object Phase2TestFailedRole extends CsrAuthorization {
     override def isAuthorized(user: CachedData)(implicit request: RequestHeader, lang: Lang) =
-      activeUserWithActiveApp(user) && statusIn(user)(PHASE2_TESTS_FAILED)
+      activeUserWithActiveApp(user) && isPhase2TestsFailed(user)
   }
 
   object Phase2TestInvitedRole extends CsrAuthorization {
     override def isAuthorized(user: CachedData)(implicit request: RequestHeader, lang: Lang) =
-      activeUserWithActiveApp(user) && statusIn(user)(PHASE2_TESTS)
+      activeUserWithActiveApp(user) && (statusIn(user)(PHASE2_TESTS) && isPhase2TestsInvited(user))
   }
 
   object Phase2TestExpiredRole extends CsrAuthorization {
     override def isAuthorized(user: CachedData)(implicit request: RequestHeader, lang: Lang) =
-      activeUserWithActiveApp(user) && statusIn(user)(PHASE2_TESTS) && isPhase2TestExpired(user)
+      activeUserWithActiveApp(user) && isPhase2TestsExpired(user)
   }
 
   object Phase3TestInvitedRole extends CsrAuthorization {
     override def isAuthorized(user: CachedData)(implicit request: RequestHeader, lang: Lang) =
-      activeUserWithActiveApp(user) && statusIn(user)(PHASE3_TESTS)
+      activeUserWithActiveApp(user) && (statusIn(user)(PHASE3_TESTS) && isPhase3TestsInvited(user))
   }
 
   object Phase3TestExpiredRole extends CsrAuthorization {
     override def isAuthorized(user: CachedData)(implicit request: RequestHeader, lang: Lang) =
-      activeUserWithActiveApp(user) && statusIn(user)(PHASE3_TESTS) && isPhase3TestExpired(user)
+      activeUserWithActiveApp(user) && isPhase3TestsExpired(user)
   }
 
   object Phase3TestFailedRole extends CsrAuthorization {
     override def isAuthorized(user: CachedData)(implicit request: RequestHeader, lang: Lang) =
-      activeUserWithActiveApp(user) && statusIn(user)(PHASE3_TESTS_FAILED)
+      isPhase3TestsFailed(user)
```

Comment:
If they are EXPORTED then they are not considered active, but we still need to know if they fit the Phase3FailedRole so that the FE makes the appropriate requests to get the PHASE3 data to populate the dashboard.